app/models/computacenter/cap_change_notifier.rb

```diff
@@ -0,0 +1,32 @@
+module Computacenter
+  module CapChangeNotifier
+    def notify_computacenter_of_cap_changes?
+      Settings.computacenter.outgoing_api.endpoint.present?
+    end
+
+    def notify_computacenter_by_email(school, device_type, new_cap_value)
+      mailer = ComputacenterMailer.with(school: school, new_cap_value: new_cap_value)
+
+      if device_type == 'std_device'
+        mailer.notify_of_devices_cap_change.deliver_later
+      else
+        mailer.notify_of_comms_cap_change.deliver_later
+      end
+    end
+
+    def update_cap_on_computacenter!(allocation_ids)
+      ids = Array(allocation_ids)
+      api_request = Computacenter::OutgoingAPI::CapUpdateRequest.new(allocation_ids: ids)
```

Comment:
one thing to note which i spotted a while back. is that despite this class accepting an array it was always used with an array of 1 element. I'm assuming it works with an array of more than one as it was built that way but as far as I can tell this is kind of "New"

Reply:
Yes, I "fixed" it so it should work with arrays